Resources/Maps/_NF/Shuttles/akupara.yml

```diff
@@ -0,0 +1,4264 @@
+  entities:
+  - uid: 357
+    components:
+    - type: Transform
+      pos: 6.5,-7.5
+      parent: 1
+- proto: MopBucketFull
+  entities:
+  - uid: 358
+    components:
+    - type: Transform
+      pos: 2.5,-9.5
+      parent: 1
+- proto: MopItem
+  entities:
+  - uid: 359
+    components:
+    - type: Transform
+      pos: 2.5,-10.5
+      parent: 1
+- proto: NitrogenCanister
+  entities:
+  - uid: 360
+    components:
+    - type: Transform
+      anchored: True
+      pos: 1.5,2.5
+      parent: 1
+    - type: Physics
+      bodyType: Static
+- proto: OxygenCanister
+  entities:
+  - uid: 361
+    components:
+    - type: Transform
+      anchored: True
+      pos: 2.5,3.5
+      parent: 1
+    - type: Physics
+      bodyType: Static
+- proto: PestSpray
+  entities:
+  - uid: 362
+    components:
+    - type: Transform
+      pos: -0.5,2.5
+      parent: 1
+- proto: PortableGeneratorSuperPacmanShuttle
+  entities:
+  - uid: 363
+    components:
+    - type: Transform
+      pos: 6.5,-9.5
+      parent: 1
+    - type: FuelGenerator
+      on: False
+    - type: Physics
+      bodyType: Static
+    - type: InsertingMaterialStorage
+- proto: PosterContrabandAmbrosiaVulgaris
+  entities:
+  - uid: 364
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: 2.5,8.5
+      parent: 1
+- proto: PosterContrabandKudzu
+  entities:
+  - uid: 365
+    components:
+    - type: Transform
+      pos: 5.5,-3.5
+      parent: 1
+- proto: PottedPlantRandom
+  entities:
+  - uid: 366
+    components:
+    - type: Transform
+      pos: 5.5,-4.5
+      parent: 1
+  - uid: 367
+    components:
+    - type: Transform
+      pos: 5.5,7.5
+      parent: 1
+- proto: Poweredlight
+  entities:
+  - uid: 368
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 1.5,-6.5
+      parent: 1
+  - uid: 369
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 1.5,7.5
+      parent: 1
+  - uid: 370
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: 3.5,11.5
+      parent: 1
+  - uid: 371
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 3.5,7.5
+      parent: 1
+- proto: PoweredLightPostSmall
+  entities:
+  - uid: 372
+    components:
+    - type: Transform
+      pos: -3.5,0.5
+      parent: 1
+- proto: PoweredlightSodium
+  entities:
+  - uid: 373
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 6.5,-1.5
+      parent: 1
+  - uid: 374
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 6.5,2.5
+      parent: 1
+- proto: PoweredSmallLight
+  entities:
+  - uid: 375
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 3.5,-12.5
+      parent: 1
+  - uid: 376
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 3.5,-8.5
+      parent: 1
+- proto: PuddleVomit
```

Comment:
Dont add any mapped trash or puddles, it gets added with automation.

Reply:
Are you referring to the trash spawners too?

Reply:
Yes, they are automated now so they are not required or needed anymore.

Reply:
All clutter removed.